### PR TITLE
Yaw and pitch module

### PR DESCRIPTION
**Pull request by thesefer**
Guess it could be done better. "Simple" yaw and pitch, nothing too fancy for now because I do not know whether you would accept it (referring to our DM conversation).

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/garden/GardenYawAndPitch.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.utils.LorenzUtils.round
+import at.hannibal2.skyhanni.utils.RenderUtils.renderStrings
+import net.minecraft.client.Minecraft
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class GardenYawAndPitch {
+    private val config get() = SkyHanniMod.feature.garden
+    private var yawandpitchDisplay = listOf<String>()
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/garden/GardenYawAndPitch.kt]: Why using a field at all if you never read it?

File: src/main/java/at/hannibal2/skyhanni/config/features/Garden.java [hunk 2]
```diff
@@ -1030,4 +1030,12 @@ public static class CustomSpeed {
 
     @Expose
     public Position cropSpeedMeterPos = new Position(278, -236, false, true);
+
+    @Expose
+    @ConfigOption(name = "Yaw / Pitch", desc = "Show Yaw and Pitch")
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/config/features/Garden.java]: Please add a point at the end of the description.

**Review by hannibal002 — CHANGES_REQUESTED** [review]
See above

File: src/main/java/at/hannibal2/skyhanni/config/features/Garden.java (hunk 2 above)
Comment (skupfer) [reply src/main/java/at/hannibal2/skyhanni/config/features/Garden.java:1035]: Second time I forgot about the dot. Added and will push it after resolving the other review.

Comment (thesefer) [reply src/main/java/at/hannibal2/skyhanni/config/features/Garden.java]: done

**Comment by jani270** [comment]
I don't really see a reason to have this in SkyHanni, as it is already present in NEU.

**Review by hannibal002 — APPROVED** [review]

**Comment by thesefer** [comment]
@jani270 Oh, you are right, didn't see because I never used it. But aren't there always overlapping features?

**Comment by hannibal002** [comment]
> I don't really see a reason to have this in SkyHanni, as it is already present in NEU.

NEU only has 2 digit precision. This was a problem for multiple users already.

**Comment by jani270** [comment]
> > I don't really see a reason to have this in SkyHanni, as it is already present in NEU.
> 
> NEU only has 2 digit precision. This was a problem for multiple users already.

Then the better solution to make the neu feature better?

**Comment by jani270** [comment]
> @jani270 Oh, you are right, didn't see because I never used it. But aren't there always overlapping features?

Well, there are always some, but i think it would be better not to do that.

**Pull request merged**